### PR TITLE
Autocomplete in inlines bugfix

### PR DESCRIPTION
**Pull request by szok**

### CODE REVIEW

**Comment by ar4s** [comment]
:ok:

File: src/ralph/admin/templates/admin/base.html [hunk 1]
```diff
@@ -68,6 +68,25 @@
       {{ media.js }}
       <!-- extra media -->
       {% block extra_scripts %}{% endblock %}
+      <script type="text/javascript">
+        (function($) {
+            $(document).ready(function() {
+              $('.add-row a').bind('click', function() {
+                var $inline = $(this).closest('table').find(
+                  '.form-row:not(#baseobjectlicence_set-empty) .autocomplete-widget'
```
Comment (mkurek) [inline src/ralph/admin/templates/admin/base.html]: this should be generic

File: src/ralph/admin/templates/admin/base.html [hunk 2]
```diff
@@ -68,6 +68,25 @@
       {{ media.js }}
       <!-- extra media -->
       {% block extra_scripts %}{% endblock %}
+      <script type="text/javascript">
+        (function($) {
+            $(document).ready(function() {
```
Comment (mkurek) [inline src/ralph/admin/templates/admin/base.html:73]: please add some comment what does this piece of code do

**Comment by mkurek** [comment]
:ok:

**Pull request merged**